Samples/MobileBuyIntegration/MobileBuyIntegration/Info.plist

```diff
@@ -26,6 +26,10 @@
 	<string>$(EMAIL)</string>
 	<key>Phone</key>
 	<string>$(PHONE)</string>
+	<key>NSLocationAlwaysUsageDescription</key>
```

Comment:
I believe [NSLocationAlwaysUsageDescription](https://developer.apple.com/documentation/bundleresources/information-property-list/nslocationalwaysandwheninuseusagedescription?changes=_9&language=objc) is deprecated as of iOS 11, and think we are targeting deployment target of 15.6 so we only need to add NSLocationWhenInUseUsageDescription

Reply:
Funnily enough it only seemed to work for me with both, but I see it has indeed [been deprecated](https://developer.apple.com/documentation/bundleresources/information-property-list/nslocationalwaysusagedescription). Lemme try again